.github/ISSUE_TEMPLATE/2-BUG.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Problem with the pub client
+about: The pub client is printing a stack trace or failing unexpectedly.
+
+---
+
+# Environment
+
+- `pub version` or `flutter packages pub version`:
+- OS version:
+- Are there any firewalls or proxies that could be impacting the problem?
```

Comment:
Might be worth including something along the lines of:

```md
Are you using the [Chinese community mirror](https://flutter.dev/community/china) or
are you sitting behind a [corporate proxy](https://www.dartlang.org/tools/pub/troubleshoot#pub-get-fails-from-behind-a-corporate-firewall).
```

Reply:
Done, I didn't use the markdown formatting for the links because when a user sees this it'll be as plain text in the issue editor, not as rendered markdown.